CryptoTechReminderSystem.Main/Program.cs

```diff
@@ -1,43 +1,97 @@
 ﻿using System;
-using System.Collections.Generic;
+using System.Threading;
 using CryptoTechReminderSystem.Boundary;
 using CryptoTechReminderSystem.Gateway;
 using CryptoTechReminderSystem.UseCase;
 using dotenv.net;
+using FluentScheduler;
 
 namespace CryptoTechReminderSystem.Main
 {
-    class Program
+    class Clock : IClock
     {
-        static void Main(string[] args)
+        public DateTimeOffset Now()
+        {
+            return DateTimeOffset.Now;
+        }
+    }
+
+    public class Program : Registry
+    {
+        private static void Main(string[] args)
         {    
             DotEnv.Config(false);
-            
-            var remindDeveloper = new SendReminder(
-                new SlackGateway(
-                    "https://slack.com/",
-                    Environment.GetEnvironmentVariable("SLACK_TOKEN")
-                )
+            JobManager.Initialize(new ReminderRegistry());
+            SpinWait.SpinUntil(() => false);
+        }
+    }
+
+    public class ReminderRegistry : Registry
+    {
+        private readonly IClock _clock;
+        private readonly GetLateDevelopers _getLateDevelopers;
+        private readonly SendReminder _sendReminder;
+
+        public ReminderRegistry()
+        {
+            var slackGateway = new SlackGateway(
+                "https://slack.com/",
+                Environment.GetEnvironmentVariable("SLACK_TOKEN")
+            );
+            var harvestGateway = new HarvestGateway(
+                "https://api.harvestapp.com/",
+                Environment.GetEnvironmentVariable("HARVEST_TOKEN"),
+                Environment.GetEnvironmentVariable("HARVEST_ACCOUNT_ID")
```

Comment:
The request header for the `UserAgent` can also be an environment variable which also needs to be passed in as an argument here.

Reply:
Added environment variable for user agent in https://github.com/chubberlisk/crypto_tech/pull/25/commits/4aa9b651c7633661a3869eed2ac13696502033a1.